src/icons/GearIcon.tsx

```diff
@@ -0,0 +1,18 @@
+import * as React from 'react'
+import Svg, { Path } from 'react-native-svg'
+import colors from 'src/styles/colors'
+
+interface Props {
+  size?: number
+  color?: string
+}
+
+const GearIcon = ({ size = 22, color = colors.black }: Props) => (
+  <Svg width={size} height={size} fill="none">
```

Comment:
for size parameter to work, a viewBox also needs to be specified

Reply:
+1